inc/sitemaps/class-author-sitemap-provider.php

```diff
@@ -126,49 +192,18 @@ public function get_sitemap_links( $type, $max_entries, $current_page ) {
 			return $links;
 		}
 
-		$steps  = $max_entries;
-		$offset = ( $current_page > 1 ) ? ( ( $current_page - 1 ) * $max_entries ) : 0;
-
-		// Initial query to fill in missing usermeta with the current timestamp.
-		$users = get_users( array(
-			'who'        => 'authors',
-			'meta_query' => array(
-				array(
-					'key'     => '_yoast_wpseo_profile_updated',
-					'value'   => 'needs-a-value-anyway', // This is ignored, but is necessary...
-					'compare' => 'NOT EXISTS',
-				),
-			),
-		) );
-
-		foreach ( $users as $user ) {
-			update_user_meta( $user->ID, '_yoast_wpseo_profile_updated', time() );
-		}
-		unset( $users, $user );
-
-		// Query for users with this meta.
-		$users = get_users( array(
-			'who'      => 'authors',
-			'offset'   => $offset,
-			'number'   => $steps,
-			'meta_key' => '_yoast_wpseo_profile_updated',
-			'orderby'  => 'meta_value_num',
-			'order'    => 'ASC',
+		$users = $this->get_users( array(
+			'offset' => ( $current_page - 1 ) * $max_entries,
+			'number' => $max_entries,
 		) );
 
-		/**
-		 * Filter the authors, included in XML sitemap.
-		 *
-		 * @param array $users Array of user objects to filter.
-		 */
-		$users = apply_filters( 'wpseo_sitemap_exclude_author', $users ); // TODO deduplicate filter. R.
+		$users = $this->exclude_users( $users );
 
 		if ( empty( $users ) ) {
 			$users = array();
 		}
 
-		// Ascending sort.
-		usort( $users, array( $this, 'user_map_sorter' ) );
+		$time = time();
```

Comment:
It's not going to make a meaningful difference on any low amount of calls, and `time()` is much less cluttered thing to read.